overview/glossary.md

```diff
@@ -0,0 +1,42 @@
+| dAPP                                                                                  | Distributed Application, generally one that relies on serverless architectures.                                                                                                                           |
+| DID                                                                                   | Decentralized Identifier used to retrieve a DID document as defined by the [W3C Community Group](https://w3c-ccg.github.io/did-spec/)                                                                     |
+| DID Document                                                                          | A DID document is an aggregation of specific types of interactions with a decentralized identifier (DID).  To see one implementation of how uPort resolves a DID document, see our [Ethr-DID-Resolver](https://github.com/uport-project/ethr-did-resolver ) library.  Other types of DID resolvers offered by uPort: [uPort-DID-Resolver](https://github.com/uport-project/uport-did-resolver),  [muPort-DID-Resolver](https://github.com/uport-project/muport-did-resolver), [Secp256k1-DID-Resolver](https://github.com/uport-project/secp256k1-did-resolver) |
+| [DID-Resolver](https://github.com/uport-project/did-resolver)                         | A library that provides a simple interface to resolve DID documents from a decentralized identifier.                                                                                                      |
+| [Eth-DID-Resolver](https://github.com/uport-project/eth-did-resolver)                 | A library that takes an ethereum address as a decentralized identifier and wraps it in a DID document.                                                                                                       |
+| [Ethereum-Claims-Registry](https://github.com/uport-project/ethereum-claims-registry) | A smart contract that manages the storage of claims (attestations) made by identities and other smart contracts on the ethereum blockchain.                                                                |
+| [Ethr-DID](https://github.com/uport-project/ethr-did)                                 | The Ethr-DID library uses ethereum addresses as Decentralized Identifiers (DIDs), allowing the easy creation and management of keys for decentralized identities (DIDs)                                                                                                                                                                                                          |                                                                                                                                                                                |
```

Comment:
The EthrDid library allows you to create Decentralized Identifiers and manage their interactions in your own app.